views/prisonerProfile/partials/prisonerProfileHeader.njk

```diff
@@ -188,20 +189,5 @@
 </div>
 
 {% block pageScripts %}
-    <script>
-      function expandLink(href) {
-        window.location.href = href
-
-        const $neuroAccordion = document.querySelector('#prisoner-accordion-heading-5')
-        $neuroAccordion.click( () => {
-          const expanded = $(this).attr('aria-expanded')
-          const heading = this.innerText
-          const action = expanded === 'false' ? 'accordion_expanded' : 'accordion_contracted'
-          gtag('event', action, {
-            event_category: 'Education',
-            event_label: heading,
-          })
-        })
-      }
-    </script>
+    <script src="/static/js/education-actions.js"></script>
```

Comment:
Do you need this - I think it is part of every page (I might be wrong)?

Reply:
Leaving in, as not specifically included in the common code related to this page